tests/lib/path_utils.py

```diff
@@ -20,3 +22,34 @@ def expand_env_vars(obj):
     else:
         obj = os.path.expandvars(obj)
     return obj
+
+
+def get_os_name():
+    """Function for getting OS name"""
+    if sys.platform == "win32":
+        os_name = 'Windows'
+    else:
+        os_name = 'Linux'
+    return os_name
+
+
+def library_name(base_name):
+    """Function for getting absolute path in OpenVINO directory to specific lib"""
+    os_name = get_os_name()
+    prefix = {'Windows': '', 'Linux': 'lib'}
+    extension = {'Windows': 'dll', 'Linux': 'so'}
+    all_libs = [{'libs': ['inference_engine_transformations', 'MKLDNNPlugin'],
+                 'path': {'Windows': Path('deployment_tools/inference_engine/bin/intel64/Release'),
+                          'Linux': Path('deployment_tools/inference_engine/lib/intel64')}},
+                {'libs': ['ngraph'],
+                 'path': {'Windows': Path('deployment_tools/ngraph/lib'),
+                          'Linux': Path('deployment_tools/ngraph/lib')}}
+                ]
+    result = ''
+    for libs in all_libs:
+        if base_name in libs['libs']:
+            result = libs['path'][os_name].joinpath(f'{prefix[os_name]}{base_name}.{extension[os_name]}')
+            break
+    assert result != '', \
```

Comment:
this is not an issue found by a test but rather a kind of infrastructure issue. Please raise exceptions for those, lets reserve assert for test conditions versification